templates/scripts/db-ops/job-create-db-postgres.yml.tpl

```diff
@@ -39,9 +38,14 @@ spec:
         image: $DOCKER_IMAGE_TAG
         command:
         - sh
-        args:
-        - '-c'
-        - psql -U$MASTER_RDS_USERNAME -h $DB_ENDPOINT postgres -v ON_ERROR_STOP=1 -a -f/db-ops/create-user.sql > /dev/null
+        - -c
+        - |
+          for db in $DB_NAME_LIST; do
+            [[ echo '\l' | psql -U$MASTER_RDS_USERNAME -h $DB_ENDPOINT postgres | grep \$db ]] || \
+            cat /db-ops/create-user.sql | \
+            sed \"s/{{ DB_NAME }}/\$db/g\" | \
```

Comment:
mysql one is called VAR_DB, postgres is DB_NAME

Reply:
fixed